### ISSUE TITLE
Research and come up with plan for link targets

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#2564]
WCAG best practice is to not have links open in a new tab unless it's necessary

https://www.w3.org/TR/WCAG20-TECHS/G200.html

**Comment by cthomasONRR** [comment]
@Maroyafaied Here's some thoughts so far. I'd be happy to do a call to discuss when you have time.

**https://www.nngroup.com/articles/new-browser-windows-and-tabs/**
* It's best to refrain from making links open in new browser windows.
* One good exception is if the user will need to refer to that content in order to complete a task in another window.
  * The example this site uses is someone doing taxes.
* They emphasize doing user research testing to see when opening a new window is justified.
* I think this justifies making our PDF docs open in a new window- usually the person will still want to reference the main onrr.gov page while also referencing the PDF.
  * I know we haven't done specific user testing on this but since most of our PDFs are reference it's reasonable to think that the user isn't done with onrr.gov after opening a PDF.
* I'm not sure if we can say the same for every external link. I think most people who navigate to onrr.gov for the eCommerce or WebCenter Portal links are done with onrr.gov, and opening in a new tab probably isn't necessary.
* We are definitely blocked with updating links until Jeff has time to update the link editor.
	
**https://designsystem.digital.gov/components/link/**
* USWDS doesn't have specific guidance about opening links in a new tab.
* They do have specific guidance for indicating if a link is external.
* We do not have external link indicators on onrr.gov or NRRD. We need to add this.
* I think marking links as external might be more important than if they are opening in a new tab.
* We will need Jeff's help to add this functionality into the inline link editor.

**https://www.w3.org/TR/WCAG20-TECHS/G200.html**
* This WCAG pretty much says the same thing as the first article (https://www.nngroup.com/articles/new-browser-windows-and-tabs/)

**Checking websites:**
* NRRD: Everything opens in the same tab (internal links, internal PDFs, external links), but none indicate if the link is external.
* Blog: I didn't check every post, but most links open in the same tab. An exception is the Plain Language blog where a reviewer asked me to make sure links opened in a new tab and I complied. Aside from this blog, I think the others are ok but should do a full audit anyway. Do we need external links on the blog to also have an indicator?
* BOEM: Internal links and internal PDFs all opened in the same tab. There were some (but not all) external links that opened in a new tab. So they have some consistency issues too. (ref: https://www.boem.gov/oil-gas-energy/leasing/ocs-lands-act-history) 
* DOI: Everything I checked (internal links, internal PDFs, external links) all opened in the same tab.
* USGS: Everything I checked (internal links, internal PDFs, external links) all opened in the same tab.
* IRS: Some internal PDFs open in same tab, some open in a new tab (ref: https://www.irs.gov/forms-instructions-and-publications). External links open in same tab & have an external indicator.
* Digital.gov: External links don't always have an indicator. All links I tested open in the same tab.


**So, I'm thinking of a few action items:**
* Possibly conduct user research on onrr.gov to see if they want PDFs and external links opening in a new tab or not.
* Do an inventory on onrr.gov of all links and indicate: 1) internal/external, and 2) opening in new tab/same tab.
* Detail the issues to Jeff so he can improve the inline link editor to: 1) easily update existing links to make sure they open in the same tab, and 2) indicate external links.
* Whatever we decide, make sure we are consistent between all sites.
